docs/integrations/enterprise-connectors/README.md

```diff
@@ -2,6 +2,6 @@
 
 Airbyte Enterprise Connectors are a selection of premium connectors available exclusively for Airbyte Self-Managed Enterprise and Airbyte Teams customers. These connectors, built and maintained by the Airbyte team, provide enhanced capabilities and support for critical enterprise systems that are not available to users of Airbyte Open Source and Airbyte Cloud. Key benefits of these connectors include support for larger data sets, parallelism for faster data transfers, and that they are covered under Airbyte Support SLAs. 
 
-If you participate in the pre-release phase of an enterprise connector, you will retain the right to use it after its main release, even if you are not a customer of Airbyte Self-Managed Enterprise, or Airbyte Teams.
+![Enterprise Connector Catalog](../../enterprise-setup/assets/enterprise-connectors-catalog.png)
```

Comment:
Is this a thing today?  I think docs should represent the current state for uses who need guidance at the moment

Reply:
It's a thing very soon (in that these are confirmed mockups). in general for enterprise it's totally okay to show the future a little bit.

Reply:
Can we delay this docs PR until that feature is released?

Reply:
I think it's completely okay to talk and show things that aren't entirely live yet in docs - especially in overview pages, and pages that do not list any explicit instructions

Reply:
ok, I'm convinced... especially as you'll be the one fielding questions :D